src/en/tutorial-client-basic.md

```diff
@@ -0,0 +1,367 @@
+Title: Basic client usage - tutorial
+TODO:  Warning: Ubuntu release versions hardcoded
+       Include links to planned tutorials (tutorial-bundles.md, tutorial-constraints.md)
+
+# Basic client usage - tutorial
+
+The goal of this tutorial is to give new Juju operators a solid introduction to
+the command line client. In so doing, several important concepts are
+introduced. It also discusses the use of command prefixes and aliases. Further
+reading suggestions are included at the end.
+
+## Prerequisites
+
+The following prerequisites are assumed as a starting point for this tutorial:
+
+ - You're using Ubuntu 18.04 LTS (Bionic).
+ - Juju (stable snap channel) is installed. See the [Installing Juju][install]
+   page.
+ - You have chosen a backing cloud and have created a controller for it. Refer
+   to the [Clouds][clouds] page to get started with a cloud and controller.
+ 
+This guide uses LXD (`v.3.9`) as a backing cloud due to its accessibility and
+low resource usage. Choose a local LXD cloud if you're not sure what to use
+(see [Using LXD with Juju][clouds-lxd]).
+
+## Command prefixes and aliases
+
+As you gain more experience with the client, you will encounter command
+prefixes whose meanings may not be immediately apparent.
```

Comment:
Starting off this section with a "you're not going to understand these" seems a bit of a harsh intro. I'd start out with the show, list, etc. Only then getting to remove and then noting the destroy vs remove vs kill (and honestly kill is only on a kill-controller so I'd leave it off of here and save it for the removing a controller section)

Reply:
I didn't say the reader won't understand it. But OK, you want something more upbeat.

Right but there is `kill-controller` as well as `destroy-controller`, and 'destroy-*' was covered.